GitTfs.Vs2015/GitTfs.Vs2015.csproj

```diff
@@ -40,9 +40,6 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
-    <Content Include="app.config.install.xdt">
```

Comment:
Please revert all the changes of this files. That's tricky but the explaination is here: https://github.com/git-tfs/git-tfs/blob/master/doc/paket.md#when-adding-a-new-package